docs/source/best-practices/homequota.rst

```diff
@@ -0,0 +1,54 @@
+*******************************************************
+Staying Under /home/<username> Quota
+*******************************************************
+It is vital to stay under your /home/<username> quota to prevent issues in using Open OnDemand and performing tasks on Discovery. Here are some suggestions to stay under quota and general best practices with using your /home/<username> directory.
+
+Utlilize /scratch and /work Directories
+=======================================================
+For long term research work storage, /work should be used. If your PI does not have space in /work setup, they can request it using the following form https://bit.ly/NURC-NewStorage and can request additional storage using https://bit.ly/NURC-StorageExtension. More details of /work can be found at :ref:`discovery_storage` For temporary job files you can utilize /scratch/<username>. Please be mindful of the purge policy of /scratch which is covered in :ref:`discovery_storage.rst`.
+
+Determine Large Files/Directories in /home/<username>
+=======================================================
+From a compute node, ``srun --pty /bin/bash``, run the following command from your /home/<username> directory ::
+    
+ du -h --max-depth=1 *
```

Comment:
will this include "hidden" files and and directories ? will it include ~/.local and ~/.conda ?

Reply:
I updated the command slightly so now it includes the hidden directories and provides the total of the /home/$USER directory too which might be helpful for them to see how close or over they are of the quota.